knowledge-base/calendar-hide-weekends.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Hide Weekends in Calendar
+description: How to hide Weekends in Calendar?
+type: how-to
+page_title: Hide Weekends in Calendar
```

Comment:
It's true that we can omit articles in titles, but here it just doesn't sound well.

```suggestion
page_title: Hide Weekends in the Calendar
```